tasks/build.ts

```diff
@@ -26,7 +26,7 @@ const StencilBuildProcess = (flags: string[]) =>
   });
 
 const DEV_FLAGS = ["--dev", "--watch", "--serve"];
-const PROD_FLAGS: string[] = [];
+const PROD_FLAGS = ["--prerender", "--debug"];
```

Comment:
do we need debug here still?

Reply:
No, but it doesn't hurt. Just incase we ever wanna debug console output